### PR TITLE
Dependabot: Add proper dependabot config file

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    groups:
+      k8sio:
+        patterns:
+        - k8s.io/*
+
+  - package-ecosystem: "github-actions"
+    target-branch: master
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (TessaIO) [inline .github/dependabot.yml:24]: Should we make this also weekly and on Sunday?

Comment (ArangoGutierrez) [reply .github/dependabot.yml:24]: This doesn't change that often as go deps. so catching a new version of an action is ok

**Comment by ArangoGutierrez** [comment]
> Thanks @ArangoGutierrez, this is a great improvement 👍 Two comments below

Ready PTAL @marquiz

File: .github/dependabot.yml (hunk 1 above)
Comment (marquiz) [inline .github/dependabot.yml:24]: I think we want the labels here, too, right

```suggestion
      interval: "daily"
    labels:
    - dependencies
    - ok-to-test
```

**Review by marquiz — APPROVED** [review]
/lgtm

**Pull request merged**